lix/packages/sdk/src/open/openLix.ts

```diff
@@ -189,7 +187,25 @@ export async function openLix(args: {
 			}
 			return addComment({ ...args, db, currentAuthor });
 		},
+		createBranch: (
+			name: string,
+			{ branchId }: { branchId: string | undefined } = { branchId: undefined },
+		) =>
```

Comment:
I recommend to always use objects as arguments to decrease the risk of breaking changes. 

- ordering of arguments is irrelevant (imagine that `name` becomes optional, then the API is screwed with `createBranch(undefined, { branchId })`)
- adding and removing arguments is possible

```suggestion
		createBranch: (args: { id?: string, name: string}) =>
```